src/collection/evaluate_collection.py

```diff
@@ -334,3 +335,175 @@ def assess_scroll_retrieval(
         )
 
         return result_ids
+
+
+def calculate_mean_values(data_list):
```

Comment:
One for later, when we have better test data - when calculating the mean values for recall and precision we should consider which mean is most appropriate. I'd argue for macro-averaged, as this would give equal weight to each class in our labels, and ignore class imbalance. Not a big issue at the moment as we're likely to improve our test set in future.